pkg/testutil/testutil.go

```diff
@@ -343,11 +343,15 @@ func (b *Base) EnsureContainerExited(con string, expectedExitCode int) {
 type Cmd struct {
 	icmd.Cmd
 	*Base
+	runResult *icmd.Result
 }
 
 func (c *Cmd) Run() *icmd.Result {
```

Comment:
Should be renamed to something like RunCached, and the existing Run should be kept uncached?

Reply:
Is there any case where we would like it to be able to run multiple times? (which is breaking things like WithStdIn, for eg)

Reply:
Alternatively I could make Run to always run, but all the assert func to call on runWithCache so that they dont trigger an exec if one has run already. Do you prefer this?

Reply:
I guess it should panic when one has run already?

Reply:
We can make it so - but I guess that would mean we would have to to fix it everywhere in the codebase where there are a succession of cmd.Assert on the same cmd object?

Reply:
Right